pkg/controller/devicetainteviction/device_taint_eviction_test.go

```diff
@@ -2375,12 +2374,8 @@ func TestEviction(t *testing.T) {
 //
 // This runs in a bubble (https://pkg.go.dev/testing/synctest), so we can wait for goroutine
 // activity to settle down and then check the state.
-func TestDeviceTaintRule(t *testing.T) {
-	tCtx := ktesting.Init(t)
-	tCtx.SyncTest("", testDeviceTaintRule)
-}
-
-func testDeviceTaintRule(tCtx ktesting.TContext) {
+func TestDeviceTaintRule(t *testing.T) { ktesting.Init(t).SyncTest("", synctestDeviceTaintRule) }
+func synctestDeviceTaintRule(tCtx ktesting.TContext) {
```

Comment:
Overall I could take or leave the `synctest...` naming convention if the `SyncTest` wrapping is nearby and explicit like this. I think this general pairing of `func TestSomething` directly followed by the actual test is easy enough to follow either way.

Reply:
I agree. I wouldn't call it out in reviews, no matter what the original author decided to do.